scripts/deploy.py

```diff
@@ -268,19 +305,53 @@ def _execute_deployment():
 
         print 'Returning to oppia/ root directory.'
 
+    # This is required to compose the release_version_library_url correctly.
+    if '.' in current_release_version:
+        raise Exception('Current release version has \'.\' character.')
+
+    release_version_library_url = (
+        'https://%s-dot-%s.appspot.com/library' %
+        current_release_version, APP_NAME)
+    answer = check_errors_in_a_page(
+        release_version_library_url, 'Library page is loading correctly?')
+    if answer in ['y', 'ye', 'yes']:
```

Comment:
I think you can do the y/ye/yes checking in check_errors_in_a_page, and have that return True or False.

Reply:
Done.